cli/chain.go

```diff
@@ -68,8 +63,8 @@ var ChainCmd = &cli.Command{
 		ChainEncodeCmd,
 		ChainDisputeSetCmd,
 		ChainPruneCmd,
-		ChainExecEVMCmd,
-		ChainInvokeEVMCmd,
+		EthDeployCmd,
```

Comment:
I think these can be removed, since they've moved to their new home?

Reply:
Oops, refactor.